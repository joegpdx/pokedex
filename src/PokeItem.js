import React, { Component } from 'react';

export default class PokeItem extends Component {
    render() {

        return (
            <li>
                <div>
                    <p>Pokemon: {this.props.pokemon.pokemon}</p>
                    <img alt="" src={this.props.pokemon.url_image} />
                    <p>Attack: {this.props.pokemon.attack}</p>
                    <p>defense: {this.props.pokemon.defense}</p>
                    <p>HP: {this.props.pokemon.hp}</p>
                </div>
            </li>
        )
    }
}