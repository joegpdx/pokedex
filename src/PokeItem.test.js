import React from 'react';
import renderer from 'react-test-renderer';
import PokeItem from './PokeItem.js';
import pokeData from './Data.js';

test('renders PokeItem.js correctly', () => {
    const tree = renderer
        .create(<PokeItem pokemon={pokeData} />)
        .toJSON();
    expect(tree).toMatchSnapshot();
});