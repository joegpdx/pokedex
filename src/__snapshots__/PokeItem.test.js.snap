// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders PokeItem.js correctly 1`] = `
<li>
  <div>
    <p>
      Pokemon: 
    </p>
    <img
      alt=""
    />
    <p>
      Attack: 
    </p>
    <p>
      defense: 
    </p>
    <p>
      HP: 
    </p>
  </div>
</li>
`;
